tests/dragonfly/cluster_test.py

```diff
@@ -142,3 +148,121 @@ async def test_cluster_nodes(async_client):
     assert info['last_ping_sent'] == '0'
     assert info['slots'] == [['0', '16383']]
     assert info['master_id'] == "-"
+
+
+@dfly_args({"proactor_threads": 4, "cluster_mode": "yes"})
+async def test_cluster(df_local_factory):
+    masters = [
+        df_local_factory.create(port=BASE_PORT+i+1, admin_port=BASE_PORT+i+1001)
+        for i in range(2)
+    ]
+
+    df_local_factory.start_all(masters)
+
+    c_masters = [aioredis.Redis(port=master.port) for master in masters]
+    c_masters_admin = [aioredis.Redis(port=master.admin_port) for master in masters]
+
+    master_ids = []
+
+    for i in range(len(masters)):
```

Comment:
```
   master_ids = await asyncio.gather(*(admin_client.execute_command("DFLYCLUSTER MYID")
                           for c_admin in c_masters_admin))
```

Reply:
Just FYI that I needed to `decode()` each ID, which slightly made this statement more complex, but I followed through with your comment